### PR TITLE
Feature/presentation area fix

### PR DESCRIPTION
**Pull request by Jigar-bhaliya**

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/Tabs/TabsContainer.jsx [hunk 1]
```diff
@@ -54,6 +83,8 @@ const TabsContainer = ({ mountModal, defaultPresentation, ...props }) => {
       ...prevState,
       [fileType]: value,
     }));
+    const currentPresentation = getPresentation(value);
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/Tabs/TabsContainer.jsx:87]: Add empty line before

File: bigbluebutton-html5/imports/ui/components/Tabs/components/Slide/SlideView.jsx [hunk 2]
```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Slide = ({ name, image }) => (
+const SlideView = ({
+  name, image, handleSlideClick, pageNum,
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/Tabs/components/Slide/SlideView.jsx]: multi line

File: bigbluebutton-html5/imports/ui/components/Tabs/components/TabPresentationContent/TabPresentationContent.jsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React, { Fragment } from 'react';
 import Select from 'react-select';
 
 import TabPositionButtonGroup from '../TabPositionButtonGroup';
-import Slide from '../Slide';
+import SlideContainer from '../Slide';
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/Tabs/components/TabPresentationContent/TabPresentationContent.jsx]: Here `import Slide from '../Slide';` is fine

File: bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+      return () => {
+        window.removeEventListener('resize', handleResize);
+      };
+    }, [myRef]);
+
+    return dimensions;
+  };
+
+  const divRef = useRef(null);
+
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx]: remove empty line

File: bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+  const svgRef = useRef(null);
+
+  const { width, height } = useContainerDimensions(divRef);
+
+  const MAX_Z_INDEX = (2 ** 31) - 1;
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx]: should be at the top below imports

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/service.js [hunk 6]
```diff
@@ -109,7 +111,8 @@ const requestPresentationUploadToken = (
   let computation = null;
   const timeout = setTimeout(() => {
     computation.stop();
-    reject({ code: 408, message: 'requestPresentationUploadToken timeout' });
+    const errorRequestObj = { code: 408, message: 'requestPresentationUploadToken timeout' };
+    reject(errorRequestObj);
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/service.js:117]: empty line before

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/service.js [hunk 7]
```diff
@@ -132,7 +135,8 @@ const requestPresentationUploadToken = (
     }
 
     if (PresentationToken.failed) {
-      reject({ code: 401, message: `requestPresentationUploadToken token ${PresentationToken.authzToken} failed` });
+      const errorPresentationToken = { code: 401, message: `requestPresentationUploadToken token ${PresentationToken.authzToken} failed` };
+      reject(errorPresentationToken);
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/service.js:144]: Add new line before

File: bigbluebutton-html5/imports/ui/components/Tabs/TabsContainer.jsx (hunk 1 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/Tabs/TabsContainer.jsx:87]: fixed

File: bigbluebutton-html5/imports/ui/components/Tabs/components/Slide/SlideView.jsx (hunk 2 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/Tabs/components/Slide/SlideView.jsx]: fixed

File: bigbluebutton-html5/imports/ui/components/Tabs/components/TabPresentationContent/TabPresentationContent.jsx (hunk 3 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/Tabs/components/TabPresentationContent/TabPresentationContent.jsx]: fixed

File: bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx (hunk 4 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx (hunk 5 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/WhiteboardWrapper/WhiteboardWrapperContainer.jsx]: fixed

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/service.js (hunk 6 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/service.js:117]: added

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/service.js (hunk 7 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/service.js:144]: added

**Pull request merged**